source/_integrations/fortigate.markdown

```diff
@@ -12,6 +12,8 @@ ha_domain: fortigate
 
 This is a FortiGate presence sensor based on device detection of the FortiGate API
 
+This integration will be deprecated in order to only use the fortios integration
```

Comment:
Please put this in a note and maybe a link to the fortios integration

Reply:
very good idea, thanks

Reply:
Can you use this format: https://developers.home-assistant.io/docs/documentation_create_page/#html

Reply:
changed it, thanks